ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue

```diff
@@ -142,7 +195,7 @@ export default defineComponent({
     })
 
     /** Clear/reset forms when select option changes. **/
-    return { addressSchema, ...toRefs(localState) }
+    return { addressSchema, addressLocal, enableAddressComplete, ...toRefs(localState) }
```

Comment:
You have no reference to `addressLocal` in the template, so i don't believe it requires returning here

Reply:
I have updated the code and address local is now being used for form.